internal/service/autoscaling/policy.go

```diff
@@ -87,6 +87,276 @@ func ResourcePolicy() *schema.Resource {
 							MaxItems: 1,
 							Elem: &schema.Resource{
 								Schema: map[string]*schema.Schema{
+									"customized_scaling_metric_specification": {
+										Type:          schema.TypeList,
+										Optional:      true,
+										MaxItems:      1,
+										ConflictsWith: []string{"predictive_scaling_configuration.0.metric_specification.0.predefined_scaling_metric_specification"},
```

Comment:
Is there a documentation reference that mentions this conflict? Perhaps we can add it as a comment above this line

Reply:
Hi @gypdtc, do you have a reference for this?

Reply:
Reference: https://docs.aws.amazon.com/autoscaling/ec2/APIReference/API_PredictiveScalingMetricSpecification.html
where it says:

`You must specify either a metric pair, or a load metric and a scaling metric individually.`

Thus users can only use a load metric which is either a predefined metric or customized metric